### DISCUSSION TITLE
Your requirements could not be resolved to an installable set of packages.

### DISCUSSION THREAD

**Discussion by macellinoss275** [livewire/livewire#6056] [7. Bugs (archived)]
### Livewire version

3

### Which PHP version are you using?

PHP 8.2

### Additional information

I just created a new repository, installed Laravel 10, and then immediately tried to install Livewire 3 but it gave me this error:

```
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Root composer.json requires livewire/livewire 3.0@beta, found livewire/livewire[dev-throw-error-when-testing-non-livewire-class, ..., dev-bad-hotfix-test, v0.0.1, ..., v0.7.4, v1.0.0, ..., 1.x-dev, v2.0.0, ..., 2.x-dev, v3.0.0-beta.1, v3.0.0-beta.2, v3.0.0-beta.3, v3.0.0-beta.4] but it does not match the constraint.       


Installation failed, reverting ./composer.json and ./composer.lock to their original content.
```

I didn't do anything else, it's literally a brand new freshly installed Laravel 10 project but it threw me this error. I'm using Windows 11 and I have already tried solutions provided by others like wrapping the composer require command inside a double quote like this:

`composer require "livewire/livewire:^3.0@beta"`

But it still didn't work. Can anyone help me please?

**Comment by joshhanley** [comment]
@macellinoss are you on windows? There are some issues with installing on Windows, see here for more details and commands you can try https://github.com/livewire/livewire/pull/6076#issuecomment-1656336235

Hope this helps!
